pkg/overlay/cache.go

```diff
@@ -273,6 +275,12 @@ func (cache *Cache) UpdateNodeInfo(ctx context.Context, node storj.NodeID, nodeI
 	return cache.db.UpdateNodeInfo(ctx, node, nodeInfo)
 }
 
+// UpdateContained updates node's contained status in node dossier
+func (cache *Cache) UpdateContained(ctx context.Context, nodeID storj.NodeID, isContained bool) (err error) {
```

Comment:
Is this a convention to use named `err` in the return value? I don't see it getting used in this function tho

Reply:
yeah it has to be named to get used by `defer mon.Task()(&ctx)(&err)`, which tracks data about what functions get called https://github.com/spacemonkeygo/monkit